src/logdriver/UserEvent.ts

```diff
@@ -12,7 +12,7 @@ export class UserEvent extends SecurityEvent {
     static readonly SESSION_TIMEOUT = new UserEvent("SESSION_TIMEOUT");
     static readonly LOCKOUT = new UserEvent("LOCKOUT");
     static readonly LOGOUT = new UserEvent("LOGOUT");
-    static readonly PERMISSIONS_CHANGE = new UserEvent("PERMISSIONS_CHANGE");
+    static readonly CHANGE_PERMISSIONS = new UserEvent("CHANGE_PERMISSIONS");
```

Comment:
Perhaps I missed a discussion, but I don't see how this is more consistent. What's the difference between 
`PERMISSIONS_CHANGE` (updated here) and `EMAIL_CHANGE` (not updated), for example? 

Maybe we should standardize on <SUBJ>_<PAST TENSE VERB>? Or maybe <SUBJ_PRESENT TENSE VERB>?

Or maybe I'm missing the point? :P

Reply:
I didn't read the ticket attached here, so now I have more context and I see the point. It might still be good to do a full pass here, but if we just want to do this one, to make them consistent across Data, Admin, and User, I'm fine with that too.

Sorry for the confusion.

Reply:
Went through and did the others - everything is VERB or VERB_OBJECT now.